website/content/development/gpio.md

```diff
@@ -85,3 +84,101 @@ At this point, we should be able to see the high/low signal on the multimeter,
 alternating between 3.3V (high) and 0V (low) every 5 seconds:
 
 <a href="/img/2020-06-15-gpio.jpg"><img src="/img/2020-06-15-gpio.thumb.jpg" srcset="/img/2020-06-15-gpio.thumb.2x.jpg 2x,/img/2020-06-15-gpio.thumb.3x.jpg 3x" width="700" style="border: 1px solid grey; margin-bottom: 2em; margin-top: 1em"></a>
+
+## Setting up an input pin
+
+To setup a Pin as an input and to act on change to High level, such as a button,
+we must poll the Pin periodically. This is needed because the
+[periph.io/device/button](https://periph.io/device/button/) example relies on
+the fact that the Raspberry Pi OS uses a custom kernel driver that enables the
+sysfs GPIO interface, which is not available on the upstream kernel.
+
+The below example configures a single Pin for Input and uses Go channels to
+signal level changes that can be acted on. Keep in mind that shortening the
+polling interval will increase CPU usage.
+
+```go
+package main
+
+import (
+	"log"
+	"time"
+
+	"periph.io/x/conn/v3/gpio"
+	"periph.io/x/conn/v3/gpio/gpioreg"
+	host "periph.io/x/host/v3"
+)
+
+type PinLevelMessage struct {
```

Comment:
This type doesn’t need to be exported. Exports in the main package don’t really do anything, either, but I think it’s good style to export as little as possible. So, change to lowercase: `type pinLevelMessage struct{`

Reply:
Of course, just an oversight. Thanks